packages/cairo.rb

```diff
@@ -10,9 +10,11 @@ class Cairo < Package
   depends_on 'fontconfig'
   depends_on 'libpng'
   depends_on 'pixman'
+  depends_on 'freetype'   # pango requires cairo with freetype
```

Comment:
You cannot install fontconfig without freetype so I don't understand your point.